src/core/components/ui/WalletBinding/index.vue

```diff
@@ -0,0 +1,191 @@
+<template lang="pug">
+  v-dialog(:value="_show", width="500", persistent)
+    v-card.pb-5
+      v-app-bar(flat, dense, color="white")
+        v-toolbar-title.title(v-if="putAccount") Metamask Wallet
+          v-spacer
+            v-btn(icon @click="closeDialog")
+              v-icon mdi-close
+            
+      div(v-if="loading")
+        div(class="mt-10 mb-10" align="center")
+          v-progress-circular(indeterminate, color="primary")
+
+      div(v-if="!loading")
+        div(v-if="putWallet")
+
+          div(align="center") Connect to your Metamask!
+            v-img(class= "mt-5 mb-3" src="@/assets/metamask-icon.png" max-width="80")
+            span(class="grey--text") Metamask
+
+          div(align="center" class="mt-5 grey--text") Haven't got a metamask wallet yet ?
+          br
+          a(href="https://docs.debio.network/getting-started")
+            u Learn How to Connect ?
+
+          div(class="mb-5 ml-8 mr-8 mt-8")
+            v-btn(
+              dark
+              class="mt-5 align-center"
+              outlined
+              width="100%"
+              large
+              elevation="2"
+              color="secondary"
+              @click="setWalllet('metamask')"
+            ) Connect
+              
+        div(class="mt-10 mb-10 ml-10 mr-10" v-if="putAccount")
+        div
```

Comment:
Note mungkin buat kedepannya karna ini kita mulai istilahnya dari awal, kita harus mulai biasain untuk engga ninggalin div sendirian kaya gini tanpa class atau id, di beberapa component debio sebelumnya ini bakal sulit di overdrive nantinya, atau kalo emang pengen gini, coba pake template jadi ga ke render nantinya di browser

CC: @ciniiia @agustinustheo @luthfiazhari @kevinjanada

Reply:
Probably she accidentally left it in there? Did you? @ciniiia

Reply:
sorry , removed mas . thank you :)